.travis.yml

```diff
@@ -4,6 +4,7 @@ python: 3.8
 
 install:
   - pip install -r requirements.txt
+  - python3 setup.py -q install
```

Comment:
Adding same setup as in vss, with that we detect any problems with setup also here